code/game/objects/items/toys.dm

```diff
@@ -952,6 +952,7 @@
 	name = "xenomorph action figure"
 	desc = "MEGA presents the new Xenos Isolated action figure! Comes complete with realistic sounds! Pull back string to use."
 	w_class = WEIGHT_CLASS_SMALL
+	item_flags = XENOMORPH_HOLDABLE
```

Comment:
They should be able to hold the `/obj/item/toy/plush/rouny` too

Reply:
wtf is a rouny

Reply:
![image](https://user-images.githubusercontent.com/54422837/210839214-79bbb2eb-639b-4a4f-87ff-97f5d9a01b34.png)

Reply:
![image](https://user-images.githubusercontent.com/41448081/210839308-2c31992c-216d-485c-a582-14cf11d67702.png)

Red beno

Reply:
rouny :)

Reply:
John you're literally a maintainer how did you not know about rouny he's straight up tgmc's mascot at this point

Reply:
what's a tgmc